### ISSUE TITLE
Ordering attributes by value changes the meaning of HTML

### ISSUE THREAD

**Issue by mvasilkov** [markedjs/html-differ#13]
Hi, I wanted to discuss the following sort() call: https://github.com/markedjs/html-differ/blob/master/lib/utils/utils.js#L24-L32

When an element has two attributes with the same name (not case-sensitive), the behavior is actually well-defined: the first attribute is used.

Consider the next example: https://codepen.io/mvasilkov/pen/abvWeqp
```html
<button style="visibility: hidden" style="display: block">
  Can't see me
</button>
```
If we change the order of the attrs we introduce a bug, yet html-differ will report identical HTML.

With the `Array#sort` function being a stable sort now (since ES2019 / Chrome 70 / Node 11 / v8 7.0), the only thing needs to be done to address this, if you choose to do so, is to drop the `a.value <> b.value` comparisons and return zero if names match.

What do you think?

**Comment by styfle** [comment]
Sounds good to me, thanks for pointing that out!

**Comment by UziTech** [comment]
LGTM @mvasilkov do you want to create a PR?

**Comment by mvasilkov** [comment]
Sure thing!

**Comment by UziTech** [comment]
Fixed in #14
